events/eid_al_fitr/meta.md

```diff
@@ -1,6 +1,6 @@
 ---
-start_date: April 9
-end_date: April 13
+start_date: March 30
```

Comment:
I see some sources claim it might begin on the 29th. There's no conflict so it's probably alright to give it another day

Reply:
Updated, and merged